app/build.gradle

```diff
@@ -27,4 +27,5 @@ dependencies {
     compile 'com.android.support:appcompat-v7:25.0.1'
     compile 'com.android.support:design:25.0.1'
     testCompile 'junit:junit:4.12'
+    compile 'com.google.android.gms:play-services-appindexing:8.4.0'
```

Comment:
This line has been added automatically by Android Studio as I used a version of API 19 for testing which includes Google Play API support. It means no harm.